docs/source/modules/cmci_action.rst

```diff
@@ -181,20 +181,37 @@ resources
 
      
   complex_filter
-    A string containing logical expressions that filter the resource table records in the data returned on the request.
+    A dictionary representing a complex filter expression. Complex filters are composed of filter expressions, represented as dictionaries. Each dictionary can specify either an attribute expression, a list of filter expressions to be composed with the ``and`` operator, or a list of filter expressions to be composed with the ``or`` operator.
+
+    The ``attribute``, ``and`` and ``or`` options are mutually exclusive with each other.
 
     Can contain one or more filters. Multiple filters must be combined using ``and`` or ``or`` logical operators.
 
     Filters can be nested. At most four nesting layers are allowed.
 
+    When supplying the ``attribute`` option, you must also supply a ``value`` for the filter.  You can also override the default operator with the ``=`` option.
```

Comment:
Should this be:
"You can also override the default operator of '=' with the `operator` option.